source/AI.cpp

```diff
@@ -2202,11 +2202,14 @@ void AI::Attack(Ship &ship, Command &command, const Ship &target)
 		return;
 	}
 
+	// Check if this ship is fast enough to keep distance from target.
+	bool isAbleToRun = target.MaxVelocity() < ship.MaxVelocity();
```

Comment:
Shouldnt we make sure its at least 10% faster or smth like that?

Reply:
```suggestion
	bool isAbleToRun = target.MaxVelocity() * 1.1 < ship.MaxVelocity();
```
like this?

Reply:
Sounds good